Podcast/DiscoverNewViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  DiscoverNewViewController.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 2/19/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class DiscoverNewViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, RecommendedSeriesTableViewCellDataSource, RecommendedSeriesTableViewCellDelegate, RecommendedTagsTableViewCellDataSource, RecommendedTagsTableViewCellDelegate, RecommendedEpisodesOuterTableViewCellDataSource, RecommendedEpisodesOuterTableViewCellDelegate {
+    
+    var tableView: UITableView!
+    
+    var series: [Series] = []
+    var tags: [String] = []
+    var episodes: [Episode] = []
+    
+    let kFooterHeight: CGFloat = 10
+    let sectionNames = ["Tags", "Series", "Episodes"]
+    let sectionHeaderHeights: [CGFloat] = [1, 32, 32]
+    let sectionContentClasses: [AnyClass] = [RecommendedTagsTableViewCell.self, RecommendedSeriesTableViewCell.self, RecommendedEpisodesOuterTableViewCell.self]
+    let sectionContentIndentifiers = ["TagsCell", "SeriesCell", "EpisodesCell"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+        view.backgroundColor = .white
+        let searchBar = UISearchController(searchResultsController: nil)
+        searchBar.searchBar.searchBarStyle = .minimal
+        searchBar.searchBar.placeholder = "Search"
+        navigationItem.titleView = searchBar.searchBar
+        searchBar.searchBar.sizeToFit()
+        searchBar.hidesNavigationBarDuringPresentation = false
+        searchBar.dimsBackgroundDuringPresentation = false
+        
+        tableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height - appDelegate.tabBarController.tabBarHeight), style: .grouped)
+        for (cls, identifier) in zip(sectionContentClasses, sectionContentIndentifiers) {
```

Comment:
why do you need zip here, why can't you just have a dictionary with identifiers as keys and classes as identifiers?